modules/clientValidation.js

```diff
@@ -16,6 +17,7 @@ function error(input, message) {
   return false;
 }
 
+// Natalie's code
```

Comment:
- [ ] Please remove comments from your code, especially those that do not explain what a function does. It would make your code cleaner.